src/pages/gen1/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -739,6 +739,74 @@ function App() {
 </Block>
 </BlockSwitcher>
 
+### Redirect URLs
+
+You can provide multiple _Sign in Redirect URI(s)_ & _Sign out redirect URI(s)_.
+
+If you have multiple redirect URI inputs, you'll need to pass them in your Amplify configuration. For example:
+
+```javascript
+Amplify.configure({
+  Auth: {
+    Cognito: {
+      loginWith: {
+        oauth: {
+          redirectSignIn: [
+            'myDevApp://'
+          ],
+          redirectSignOut: [
+            'myDevApp://',
+            'myProdApp://'
+          ],
+          ...oauthConfig
+        }
+      },
+      ...userPoolConfig
+    }
+  }
+});
+```
+
+#### Option to specify a redirect URL upon signOut
+If you have multiple redirect urls configured, you may choose to override the default behavior of selecting a redirect url and provide the one of your choosing when calling `signOut`. The provided redirect url should match at least one of the configured redirect urls. If no redirect url is provided to `signOut`, the first item from the the configured redirect urls list that does not contain a http nor https prefix will be picked.
+
+```ts
+import { Amplify } from 'aws-amplify';
+import { signOut } from 'aws-amplify/auth';
+
+Amplify.configure({
+  Auth: {
+    Cognito: {
+      loginWith: {
+        oauth: {
+          redirectSignIn: [
+            'myDevApp://'
+          ],
+          redirectSignOut: [
+            'myDevApp://',
+            'myProdApp://'
+          ],
+          ...oauthConfig
+        }
+      },
+      ...userPoolConfig
+    }
+  }
+});
+
+function handleSignOut() {
+  signOut({
+    global: false,
+    oauth: {
+      redirectUrl: 'myDevApp://'
```

Comment:
Would `myProdApp://` be a better example here as `myDevApp://` would get selected by default anyway?

Reply:
I think a better example would be to use a different url altogether so updated it to show that.